pkg/lib/saml/samlprotocol/schema.go

```diff
@@ -502,7 +517,7 @@ func (r *Response) Element() *etree.Element {
 	// cannonicalizer, but prefix lists do not appear to be implemented correctly
 	// in some libraries, so the safest action is to always produce XML that is
 	// (a) in canonical form and (b) does not require prefix lists.
-	el.CreateAttr(fmt.Sprintf("xmlns:%s", xmlSchemaNamespace), "http://www.w3.org/2001/XMLSchema")
+	el.CreateAttr("xmlns:xs", "http://www.w3.org/2001/XMLSchema")
```

Comment:
I think this change is not needed, using `xmlSchemaNamespace` here ensures we use the same namespace name in our code.

Reply:
Fixed up this line.